sdk/go/common/resource/plugin/analyzer_plugin.go

```diff
@@ -104,7 +104,11 @@ func NewPolicyAnalyzer(
 	pluginPath, err := workspace.GetPluginPath(ctx.Diag,
 		workspace.AnalyzerPlugin, policyAnalyzerName, nil, host.GetProjectPlugins())
 	if err != nil {
-		return nil, rpcerror.Convert(err)
```

Comment:
This would fail the conversion and then fail an assert.

Reply:
I wouldn't expect GetPluginPath to _ever_ return an rpcerror. I think this can just be changed to `return nil, err`.

Reply:
Also isn't the error below what we really want to see? "could not start policy pack %q because the built-in analyzer..."

Reply:
> Also isn't the error below what we really want to see? "could not start policy pack %q because the built-in analyzer..."

There are a few different types of errors that can come out of `workspace.GetPluginPath`. I don't think we can just collapse them all to this error message where the reasoning in the message is always applicable.

I don't know what the case where `err == nil && pluginPath == ""`, but I don't want to introduce any bugs from removing a guard clause so I'd like to leave it.

Reply:
> I don't know what the case where err == nil && pluginPath == "", but I don't want to introduce any bugs from removing a guard clause so I'd like to leave it.

I don't think we ever return a nil err and empty path.

> There are a few different types of errors that can come out of workspace.GetPluginPath. I don't think we can just collapse them all to this error message where the reasoning in the message is always applicable.

Sure, if you want to be conservative check for `MissingError` which is what we return when we couldn't find a path.